tests/lib/rules/class-methods-use-this.js

```diff
@@ -244,8 +244,40 @@ ruleTesterTypeScript.run("class-methods-use-this", rule, {
         { code: "class A { #bar() {} }", options: [{ exceptMethods: ["#bar"] }] },
         { code: "class A { foo = function () {} }", options: [{ enforceForClassFields: false }] },
         { code: "class A { foo = () => {} }", options: [{ enforceForClassFields: false }] },
+        { code: "class A { override foo = () => {} }", options: [{ enforceForClassFields: false }] },
```

Comment:
Can we also add test cases for [privateIdentifiers using `#`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Private_properties)